public/index.html

```diff
@@ -16,6 +16,7 @@
         </div>
       </div>
     </main>
+    <script src="/bower_components/fxa-relier-client/fxa-relier-client.min.js"></script>
```

Comment:
It seemed natural in this patch to allow a user sign in for the main buy flow if they happen to land on the URL without an access token (by editing the URL or whatever). The sign in part works but immediately after that the buy flow fails on the console because there isn't a product ID in the query string. We could improve that error later I suppose. I toyed with disallowing a sign-in on the buy flow but it introduced a lot of extra code.